src/app/workflow/launch-third-party/launch-third-party.component.spec.ts

```diff
@@ -56,14 +56,16 @@ describe('LaunchThirdPartyComponent', () => {
     const nativeElement: HTMLElement = fixture.nativeElement;
 
     // Verify urls are correct; got these from prod (except for Terra, which is new) to verify there is no breakage.
-    // tslint:disable:max-line-length
-    if (!Dockstore.FEATURES.enableMultiCloudLaunchWithDNAstack) {
-      expect(
-        nativeElement.querySelector(
-          'a[href="https://app.dnastack.com/#/app/workflow/import/dockstore?descriptorType=wdl&path=github.com/DataBiosphere/topmed-workflows/UM_aligner_wdl"]'
-        )
-      ).toBeTruthy();
-    }
+    // This test was removed as part of: https://ucsc-cgl.atlassian.net/browse/SEAB-3800
+    // /* eslint-disable max-len */
```

Comment:
hotfix branch doesn't use eslint, not sure why it's changed here

Reply:
Ack, my fault, I initially targeted the branch at develop and cherry-picked over to the hotfix. Fixed: https://github.com/dockstore/dockstore-ui2/pull/1438/commits/37ab8feae38b63ad8fb7cde5cbc25686648e250f